forc/src/cli/shared.rs

```diff
@@ -115,6 +116,11 @@ pub struct Pkg {
     /// Outputs json abi with callpaths instead of names for struct and enums.
     #[clap(long)]
     pub json_abi_with_callpaths: bool,
+    /// The IPFS Node to use for fetching IPFS sources.
+    ///
+    /// Possible values: PUBLIC, LOCAL, <GATEWAY_URL>
+    #[clap(long)]
+    pub ipfs_node: Option<IPFSNode>,
```

Comment:
Is there a way we can define this CLI arg once and have it be part of the CLI for `build`, `check`, etc?

Reply:
That is the main reasoning behind using `shared.rs` but some of the downstream tooling does not use that, will open a separate issue for that since it is not only for this field but other common fields are also repeated in some downstream tooling.

Reply:
Opened #4684